config/development/config.yaml

```diff
@@ -4,7 +4,7 @@ enableGitInfo: true
 
 # default to just building english on local dev for a faster build
 disableLanguages:
-  - fr
-  - ja
-  - ko
-  - es
+  # - fr
+  # - ja
+  # - ko
+  # - es
```

Comment:
I think we probably want this back to disabling by default before merging

Reply:
Done, thanks for the catch!